src/main/frontend/handler/page.cljs

```diff
@@ -792,10 +792,6 @@
       (fn [chosen _click?]
         (state/clear-editor-action!)
         (let [wrapped? (= page-ref/left-brackets (gp-util/safe-subs edit-content (- pos 2) pos))
-              prefix (str (t :new-page) ": ")
-              chosen (if (string/starts-with? chosen prefix) ;; FIXME: What if a page named "New page: XXX"?
-                       (string/replace-first chosen prefix "")
-                       chosen)
```

Comment:
This should fix various issues related to autocompleting page names starting with "New page" or parts of it (depends on the active language). For example, typing `[[New]]` would highlight the wrong `New` on the autocomplete component. Creating a reference starting with `New page: ` could also lead to unexpected behavior.

The new page label is now handled on the custom item renderer of `page-search`
https://github.com/logseq/logseq/pull/9610/commits/135e3228b6facdcb8541086e4a99c41655709782#diff-804cdbc15af06aba05e572b8dcee83da9a495773a899aa35f8ce42bbaa6afce0R157-R159
Note that this has a minor performance cost, because we are checking if the page exists, for each

Reply:
Nice that it fixes some issues but bummer that it adds another entity lookup call per item rendered. When I type within `[[]]` in lambda, I think the initial typing feels less responsive. If users notice then we can try to improve this